source/index.js

```diff
@@ -63,7 +63,10 @@ module.exports = async (input = 'patch', options) => {
 	let isPublished = false;
 
 	const rollback = onetime(async () => {
-		console.log('\nPublish failed. Rolling back to the previous state…');
+		console.log(); // Newline for readability
```

Comment:
You can keep the `console.log` the way it was, without splitting it.
Also, if `np` is running in preview mode, it should immediately `return` from this function in any case.

Reply:
Without new line: (Strange)
![image](https://user-images.githubusercontent.com/5731176/57185416-536db700-6ea1-11e9-8123-c3ddecb55514.png)
With new line: (Beautiful)
![image](https://user-images.githubusercontent.com/5731176/57185404-2de0ad80-6ea1-11e9-97ef-e148a32e15bb.png)

Reply:
> Without new line: (Strange)

Did you revert line 66 to be:

```js
console.log('\nPublish failed. Rolling back to the previous state…');
```

Reply:
Also, this should be reverted anyways because of the 2nd sentence in my original comment.

Reply:
@itaisteinherz what do you think about this?
```js
let status = 'failed'
if (options.preview) {
	status = 'not realized'
}
		
console.log(`\nPublish ${status}. Rolling back to the previous state…`);
```

Reply:
This can be simplified to:

```js
if (options.preview) {
	return;
}

console.log('\nPublish failed. Rolling back to the previous state…');
```